deploy/rbac.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: experiment-sa
+  labels:
+    name: experiment-sa
+---
+# Source: openebs/templates/clusterrole.yaml
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: experiment
+  labels:
+    name: experiment
+rules:
+- apiGroups: ["*"]
```

Comment:
since this is for representative purposes, shall we put a list of apigroups, resources and verbs rather than have `*/*/*`. Reference: https://github.com/litmuschaos/community/blob/master/feature-demos/chaos-operator-workflow/nginx/rbac.yaml